bpfman/src/types.rs

```diff
@@ -1958,6 +1959,49 @@ impl std::fmt::Display for MapType {
     }
 }
 
+impl From<AyaMapType> for MapType {
```

Comment:
👀 why?

Reply:
Pretty much the same reason for both AyaMapType and AyaProgramType.  Aya used to return the kernel id (u32) for both, but now it returns an enum and gives us no way to get at the id.  So, I mappped from the Aya type to the bpfman type, and then I can convert to either a u32 or string using our existing code.

Reply:
And, Rust wouldn't let me map directly from the Aya type to a u32 or string directly because we don't define either.

Reply:
> Pretty much the same reason for both AyaMapType and AyaProgramType. Aya used to return the kernel id (u32) for both, but now it returns an enum and gives us no way to get at the id. So, I mappped from the Aya type to the bpfman type, and then I can convert to either a u32 or string using our existing code.

ugh yeah iirc this is because we need to handle the u32s in the backend - i.e store to database.
I guess this is fine for now, but I think we owe a follow up PR in aya to allow conversion into u32 somehow.